webplugin/js/app/kommunicate-ui.js

```diff
@@ -827,6 +828,11 @@ KommunicateUI = {
             .addClass('n-vis');
         $applozic('#km-faqdiv').removeClass('vis').addClass('n-vis');
         $applozic('#mck-msg-new').attr('disabled', false);
+        for(let i=MCK_EVENT_HISTORY.length - 1; i>=0; i--){
+            if(MCK_EVENT_HISTORY[i].includes('faq')){
+                MCK_EVENT_HISTORY.splice(i, 1);
```

Comment:
will there be multiple entries for faq? do we need to loop through all elements, or can break here?

Reply:
yes there will can be multiple entries for faq related events. We need to remove every such event once faq closes.